src/entry/logEntry.d.ts

```diff
@@ -0,0 +1,30 @@
+import { UUID } from "node:crypto";
```

Comment:
enums are not allowed in .d.ts files
please rename this file to logEntry.ts or don't use enums

[alternatives](https://stackoverflow.com/questions/38335668/how-to-refer-to-typescript-enum-in-d-ts-file-when-using-amd)